<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NozzleClean</class>
 <widget class="QWidget" name="NozzleClean">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1101</width>
    <height>666</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NozzleClean</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(72,61, 139);</string>
  </property>
  <widget class="QPushButton" name="openCleanPushButton">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>420</y>
     <width>210</width>
     <height>90</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 18pt &quot;楷体&quot;;
background-color: rgb(0,0,230);</string>
   </property>
   <property name="text">
    <string>启用清洁液</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_1">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>110</y>
     <width>85</width>
     <height>85</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(255, 255, 255);
background-color: rgb(0, 0, 230);
font: 18pt &quot;楷体&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>110</y>
     <width>85</width>
     <height>85</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(255, 255, 255);
background-color: rgb(0, 0, 230);
font: 18pt &quot;楷体&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>110</y>
     <width>85</width>
     <height>85</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(255, 255, 255);
background-color: rgb(0, 0, 230);
font: 18pt &quot;楷体&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>110</y>
     <width>85</width>
     <height>85</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(255, 255, 255);
background-color: rgb(0, 0, 230);
font: 18pt &quot;楷体&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>110</y>
     <width>85</width>
     <height>85</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(255, 255, 255);
background-color: rgb(0, 0, 230);
font: 18pt &quot;楷体&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_6">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>110</y>
     <width>85</width>
     <height>85</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(255, 255, 255);
background-color: rgb(0, 0, 230);
font: 18pt &quot;楷体&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_7">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>110</y>
     <width>85</width>
     <height>85</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(255, 255, 255);
background-color: rgb(0, 0, 230);
font: 18pt &quot;楷体&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_8">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>110</y>
     <width>85</width>
     <height>85</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(255, 255, 255);
background-color: rgb(0, 0, 230);
font: 18pt &quot;楷体&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_10">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>240</y>
     <width>85</width>
     <height>85</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(255, 255, 255);
background-color: rgb(0, 0, 230);
font: 18pt &quot;楷体&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>10</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_14">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>240</y>
     <width>85</width>
     <height>85</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(255, 255, 255);
background-color: rgb(0, 0, 230);
font: 18pt &quot;楷体&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>14</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_15">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>240</y>
     <width>85</width>
     <height>85</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(255, 255, 255);
background-color: rgb(0, 0, 230);
font: 18pt &quot;楷体&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>15</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_12">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>240</y>
     <width>85</width>
     <height>85</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(255, 255, 255);
background-color: rgb(0, 0, 230);
font: 18pt &quot;楷体&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>12</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_13">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>240</y>
     <width>85</width>
     <height>85</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(255, 255, 255);
background-color: rgb(0, 0, 230);
font: 18pt &quot;楷体&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>13</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_9">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>240</y>
     <width>85</width>
     <height>85</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(255, 255, 255);
background-color: rgb(0, 0, 230);
font: 18pt &quot;楷体&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_11">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>240</y>
     <width>85</width>
     <height>85</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(255, 255, 255);
background-color: rgb(0, 0, 230);
font: 18pt &quot;楷体&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>11</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_16">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>240</y>
     <width>85</width>
     <height>85</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(255, 255, 255);
background-color: rgb(0, 0, 230);
font: 18pt &quot;楷体&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>16</string>
   </property>
  </widget>
  <widget class="QPushButton" name="singleCleanPushButton">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>420</y>
     <width>180</width>
     <height>90</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>单点打通</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="allCleanPushButton">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>420</y>
     <width>180</width>
     <height>90</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>整体打通</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exitButton">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>550</y>
     <width>120</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 14pt &quot;楷体&quot;;</string>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QLabel" name="previewLab">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1101</width>
     <height>666</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(77,61, 139);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <zorder>previewLab</zorder>
  <zorder>openCleanPushButton</zorder>
  <zorder>pushButton_1</zorder>
  <zorder>pushButton_2</zorder>
  <zorder>pushButton_3</zorder>
  <zorder>pushButton_4</zorder>
  <zorder>pushButton_5</zorder>
  <zorder>pushButton_6</zorder>
  <zorder>pushButton_7</zorder>
  <zorder>pushButton_8</zorder>
  <zorder>pushButton_10</zorder>
  <zorder>pushButton_14</zorder>
  <zorder>pushButton_15</zorder>
  <zorder>pushButton_12</zorder>
  <zorder>pushButton_13</zorder>
  <zorder>pushButton_9</zorder>
  <zorder>pushButton_11</zorder>
  <zorder>pushButton_16</zorder>
  <zorder>singleCleanPushButton</zorder>
  <zorder>allCleanPushButton</zorder>
  <zorder>exitButton</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
